packages/teams-js/src/public/sharing.ts

```diff
@@ -194,4 +194,80 @@ export namespace sharing {
   export function isSupported(): boolean {
     return ensureInitialized(runtime) && runtime.supports.sharing ? true : false;
   }
+
+  /**
+   * Namespace to get the list of content shared in a context
+   *
+   * @beta
+   */
+  export namespace history {
+    /**
+     * Represents IContentResponse parameters.
+     *
+     * @beta
+     */
+    export interface IContentResponse {
+      /** Id of the app where the content was shared from */
+      appId: string;
+      /** Title of the shared content */
+      title: string;
+      /** Reference of the shared content */
+      contentReference: string;
+      /** Id of the thread where the content was shared */
+      threadId: string;
+      /** Id of the user who shared the content */
+      author: string;
+      /** Type of the shared content */
+      contentType: string;
+    }
+
+    /**
+     * getContent callback function type
+     *
+     * @beta
+     */
+    export type getContentCallbackFunctionType = (
+      error: SdkError | null,
+      contentDetails: IContentResponse[] | null,
+    ) => void;
+
+    /**
+     * Get the list of content shared in a context
+     * @param callback - Callback contains 2 parameters, `error` and `contentDetails`.
+     * `error` can either contain an error of type `SdkError`, in case of an error, or null when fetch is successful.
+     * `contentDetails` will be the list of contents {@link IContentResponse} that was shared in the context, or null when the request fails.
+     *
+     * @beta
+     */
+    export function getContent(callback: getContentCallbackFunctionType): void {
+      if (!callback) {
+        throw new Error('[get content] Callback cannot be null');
+      }
+      ensureInitialized(runtime, FrameContexts.sidePanel, FrameContexts.meetingStage);
+      if (!isSupported()) {
+        throw errorNotSupportedOnPlatform;
+      }
+      sendMessageToParent(
+        getApiVersionTag(sharingTelemetryVersionNumber_v1, ApiName.Sharing_History_GetContent),
```

Comment:
>sharingTelemetryVersionNumber_v1 [](http://example.com/codeflow?start=25&length=32)

I actually don't know how these version numbers are supposed to be set. @KangxuanYe what's the right constant to use for new functions? Is it v1 like this?

Reply:
This is V2 rather than V1. Only API exists in Teams-JS library v1 and does NOT exist in Teams-JS library v2 will be labelled as V1. Vice versa.

This new API is designed to be in Teams-JS library V2 so it should fix here from v1 to v2.